app/src/main/java/org/fossasia/badgemagic/ui/fragments/main_textart/MainTextArtFragment.kt

```diff
@@ -69,6 +70,8 @@ class MainTextArtFragment : BaseFragment() {
     private val modeAdapter = ModeAdapter()
     private lateinit var viewModel: MainTextArtViewModel
 
+    private val resourceHelper = Resource()
```

Comment:
Only to be used in ViewModel. Revert to context

Reply:
Sure